src/plugins/gui/providers/kadoProvider.ts

```diff
@@ -533,12 +528,15 @@ export const kadoProvider: FiatProviderFactory = {
           throw new FiatProviderError({ providerId, errorType: 'paymentUnsupported' })
         }
 
+        const reverse = (direction === 'sell' && amountType === 'fiat') || (direction === 'buy' && amountType === 'crypto')
```

Comment:
Wouldn't the inverse of a typical quote be to "buy fiat" or "sell crypto"  (e.g. crypto -> fiat).

Reply:
normal quotes let you specify fiat amount to pay for a buy and crypto amount to receive for sell

reverse means

specify the crypto amount to buy and the fiat amount to sell.